### ISSUE TITLE
Option to disable old CMD+G shortcut

### ISSUE THREAD

**Issue by mkubenka** [foyer-work/ooogle-it#29]
Can you please add option to disable old CMD+G shortcut which now shows deprecation message. [CMD+G is reserved](https://support.apple.com/en-us/HT201236) for "Find next" / "Find again" on macOS:

> Command-G: Find Again: Find the next occurrence of the item previously found. To find the previous occurrence, press Shift-Command-G.

**Comment by siddsax** [comment]
Yes @mkubenka, we'll completely remove it in the next version for the same reason.

**Comment by mkubenka** [comment]
@siddsax Any update please?

**Comment by maebert** [comment]
+1 — this issue is PREVENTING ME FROM USING MERLIN regularly. It completely breaks my workflow so I switch Merlin off until I actually need it.

**Comment by siddsax** [comment]
@maebert @mkubenka we submitted the change to the chrome store, it should be live in a day or two. Thanks for your patience! We mostly had to delay it due to the influx of users from our initial content where cmd/ctrl+G was present.
